### PR TITLE
Stoofvlees met frietjes

### PR DESCRIPTION
**Pull request by Innen**

### CODE REVIEW

**Review by LilyAnderssonLee — APPROVED** [review]
Good to go!

File: docs/Frietjes.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Ingredients
+(loskokende) aardappel 1 kg
```
Comment (robdyr) [inline docs/Frietjes.md:2]: Perhaps we can increase the quantity?

**Review by robdyr — COMMENTED** [review]
Suggestion for quantity

File: docs/Frietjes.md (hunk 1 above)
Comment (Innen) [reply docs/Frietjes.md:2]: Always, the more the merrier. I'll double the amount of potatoes for french fries.

**Pull request merged**